tests/README.md

```diff
@@ -18,10 +18,20 @@ Travis (https://travis-ci.org/) is used to automatically build and run tests on
 
 Since Travis cannot `clasp login`, a `.clasprc.json` file is included that was created locally using `clasp login`.
 
-It is encrypted as `.clasprc.json.enc` and then decrypted by Travis:
+To then encrypt the `.clasprc.json` file, use these commands:
```

Comment:
Add a link to the installing `travis` guide:

To then encrypt the `.clasprc.json` file, use these commands using the [Travis CLI](https://github.com/travis-ci/travis.rb):